presto-docker/bin/entrypoint

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -xeuo pipefail
+
+# Copy config files, if they exist, to /usr/lib/presto/etc. This is necessary
+# for Kubernetes as Secrets and ConfigMaps are mounted read only.
```

Comment:
Actually there was an intentional change a while ago (see: https://github.com/kubernetes/kubernetes/pull/58720) which mounts ConfigMaps, Secrets, downwardAPI and projected volumes as read only. This has nothing to do with the VolumeMount and everything to do with creating consistent and predictable behavior. Specifically if you dig into the PR and related issue this change was made so that it was obvious that changes to a mounted ConfigMap, Secret, etc. did not propagate back to the original object.